testing/test_functions.py

```diff
@@ -47,3 +50,18 @@ def test_format_version(version, monkeypatch, scheme, expected):
         version,
         version_scheme=vs,
         local_scheme=ls) == expected
+
+
+@pytest.fixture
+def tmpfoo(request):
```

Comment:
good start,

please use the `tmpdir` fixture as base,
you can then use `tmpdir.strpath` as `root` and a constant name for the `write_to` variable